tests/acceptance/features/bootstrap/EncryptionContext.php

```diff
@@ -48,6 +49,101 @@ public function recreateMasterKeyUsingOccCommand() {
 		$this->featureContext->theCommandShouldHaveBeenSuccessful();
 	}
 
+	/**
+	 * @Given encryption has been enabled
+	 *
+	 * @return void
+	 */
+	public function encryptionHasBeenEnabled() {
+		$this->featureContext->runOcc(['encryption:enable']);
+	}
+
+	/**
+	 * @When the administrator sets the encryption type to :encryptionType using the occ command
+	 * @Given the administrator has set the encryption type to :encryptionType
+	 *
+	 * @param string $encryptionType
+	 *
+	 * @return void
+	 * @throws \Exception
+	 */
+	public function theAdministratorSetsEncryptionTypeToUsingTheOccCommand($encryptionType) {
+		$this->featureContext->runOcc(
+			["encryption:select-encryption-type", $encryptionType, "-y"]
+		);
+	}
+
+	/**
+	 * @When the administrator encrypts all data using the occ command
+	 * @Given the administrator has encrypted all the data
+	 *
+	 * @return void
+	 */
+	public function theAdministratorEncryptsAllDataUsingTheOccCommand() {
+		$this->featureContext->runOcc(["encryption:encrypt-all", "-n"]);
+	}
+
+	/**
+	 * @When the administrator decrypts user keys based encryption with recovery key :recoveryKey using the occ command
+	 *
+	 * @param string $recoveryKey
+	 *
+	 * @return void
+	 */
+	public function theAdministratorDecryptsUserKeysBasedEncryptionWithRecoveryKeyUsingTheOccCommand($recoveryKey) {
+		$this->featureContext->invokingTheCommandWithEnvVariable(
+			"encryption:decrypt-all -m recovery -c yes",
+			'OC_RECOVERY_PASSWORD',
+			$recoveryKey
+		);
+	}
+
+	/**
+	 * @Then the file :fileName of user :username should not be encrypted
+	 *
+	 * @param string $fileName
+	 * @param string $username
+	 *
+	 * @return void
+	 */
+	public function theFileOfUserShouldNotBeEncrypted($fileName, $username) {
+		$filePath = "data/$username/files/$fileName";
+		$this->featureContext->readFileInServerRoot($filePath);
+
+		$response = $this->featureContext->getResponse();
+		$parsedResponse = HttpRequestHelper::getResponseXml($response);
+		$encodedFileContent = (string)$parsedResponse->data->element->contentUrlEncoded;
+		$fileContent = urldecode($encodedFileContent);
+
+		PHPUnit_Framework_Assert::assertContains(
+			"ownCloud test text file 0",
```

Comment:
this text should not be hardcoded
this will fail if using an other file

Reply:
Maybe do like in ``downloadFile.feature``
```
Then the downloaded content should be "ownCloud test text file 0" plus end-of-line
```
with something like:
```
Then the file :fileName of user :username should have unencrypted content :content plus end-of-line
```
and that exposes the expected content in the feature file.